### ISSUE TITLE
[Question] Connection to a websocket server with external client

### ISSUE THREAD

**Issue by gabrielkotev** [aspnet/WebSockets#146]
Hi,
Is it possible to connect to a asp.net core websocket server with a client that is outside the solution ?
I tried to connect to the server with the same client that was inside the wwwroot folder, without any success . Also I tried to expose the IIS express to my home private network (to prevent possible loopbacks), and it worked for http request but still not for the wobsocket request.
I also tried to deploy it to an external hosting, still without any success.
I need to mention that I tried my client with a connection to ws://echo.websocket.org and it worked like a charm.
You are my only hope, and thanks in advance!

**Comment by Tratcher** [comment]
WebSockets has no addition security requirements beyond those of HTTP. What errors do you get?

**Comment by gabrielkotev** [comment]
After some research it worked.
But I will share, to spare the time of others on same issue.
I believe that it's an issue of old asp.net core version, 
I had no error, it simply didn't connect to the server.
When I changed the project to an newer asp.net core version it didn't worked once again because 
I tested it with Microsoft edge - which it was a big mistake!
Microsoft edge do not allow loopbacks!
I am sorry for the spam.
